SensorStub.cpp

```diff
@@ -0,0 +1,24 @@
+#include "SensorStub.h"
+
+SensorStub::SensorStub(int temp, int precip, int humidity, int windSpeed) {

```

Comment:
Good way to make a data-driven stub 👍 
`temp` does mean temperature, but it has ambiguity with a temp variable as well.
Also, `this->` is not required. Any reason for its usage?